### DISCUSSION TITLE
how can i make connection betwwen two models

### DISCUSSION THREAD

**Discussion by KapilBagdi** [Laravel-Backpack/community-forum#419] [Q&A (Help)] [answered]
how can i make connection between two models ihave table name employe and amployee attendance i  want to make relation between  this two table by connecting ists id

**Comment by pxpm** [comment, marked as answer]
Hey @KapilBagdi have a look at https://laravel.com/docs/10.x/eloquent-relationships#main-content to learn how to properly setup your relationship in your models. 

After that you can use the Backpack fields to handle those relations in the form interface. 

You can also check our demo repository https://github.com/Laravel-Backpack/demo, we have pretty much examples of all relationships there. 

Cheers

**Reply by KapilBagdi** [reply]
thank u for ur valuable reply @pxpm
